intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_files.h

```diff
@@ -211,9 +211,20 @@ struct model_file_loader {
     file.read_raw(&hparams.alibi_bias_max, sizeof(float));
     file.read_raw(&hparams.clip_qkv, sizeof(float));
     hparams.par_res = file.read_u32();
+
     hparams.word_embed_proj_dim = file.read_u32();
     hparams.do_layer_norm_before = bool(file.read_u32());
+    
+    // For ChatGLM-1 & 2
+    hparams.bos_token_id = file.read_u32();
+    hparams.eos_token_id = file.read_u32();
+    hparams.pad_token_id = file.read_u32();
+    hparams.sep_token_id = file.read_u32();
+    hparams.multi_query_group_num = file.read_u32();
+    hparams.ffn_hidden_size = file.read_u32();
+    hparams.inner_hidden_size = file.read_u32();
```

Comment:
Is it == `n_embed`?

Reply:
what does it mean?

Reply:
I saw you set `0` for `inner_hidden_size`.

Reply:
ChatGLM-2 requires theses:
    hparams.multi_query_group_num = file.read_u32();
    hparams.ffn_hidden_size = file.read_u32();

ChatGLM-1 requires hparams.inner_hidden_size.

I will double check whether inner_hidden_size is the same as n_embed.